src/Solver.cpp

```diff
@@ -624,11 +625,14 @@ void Solver::countFissionableFSRs() {
   log_printf(INFO, "Counting fissionable FSRs...");
 
   /* Count the number of fissionable FSRs */
-  _num_fissionable_FSRs = 0;
+  int num_fissionable_FSRs;
+#pragma omp parallel for reduction(+:num_fissionable_FSRs)
```

Comment:
Wow, so is this "smart" enough to only reduce when the conditional evaluates as `true`? I never realized this was possible. The skeptic in me just wants a confirmation that the value is the same with or without OpenMP - though I assume this is reflected in the test suite results when run with or without threading.